draft-ietf-tls-dtls13.md

```diff
@@ -531,6 +534,16 @@ packet  -->  |   OCT == 25   -+--> DTLSCipherText with CID (DTLS 1.2)
 ~~~
 {: #demux title="Demultiplexing DTLS 1.2 and DTLS 1.3 Records"}
 
+Note: The optimized DTLS header format shown in {{cid_hdr}}, which 
+does not carry the Content Type in the Unified Header format, requires 
+a different demultilexing strategy compared to what was used in previous 
+DTLS versions where the Content Type was conveyed in every record. 
+As described in {{demux}}, the first byte determines how an incoming 
+DTLS record is demultiplexed. The first 3 bits of the first byte 
+distinguish a DTLS 1.3 encrypted record from a record type used in 
+previous DTLS versions or from plaintext DTLS 1.3 record. Hence, the 
+range 32 (0b0010 0000) to 63 (0b0011 1111) needs to be excluded 
+from future allocations by IANA to avoid demultiplexing problems.
```

Comment:
```suggestion
from future allocations by IANA to avoid demultiplexing problems;
see {{iana-considerations}}.
```

(The iana-considerations anchor isn't in the doc yet, though)